src/components/alert/index.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { FC, ReactNode } from 'react';
+
+import {
+  AlertDescription,
+  AlertIcon,
+  AlertTitle,
+  Alert as ChackraAlert,
+  Flex,
+} from '@chakra-ui/react';
+
+import Link from '../link';
+
+interface Props {
+  title?: string;
+  description: string;
+  link?: {
+    text: string;
+    url: string;
+  };
+  typeMessage?: 'error' | 'warning' | 'info' | 'success';
+  icon: ReactNode;
+}
+
+const Alert: FC<Props> = ({ title, description, link, typeMessage, icon }) => {
+  return (
+    <ChackraAlert status={typeMessage}>
+      <AlertIcon>{icon}</AlertIcon>
+      <Flex direction="column">
+        {title ? <AlertTitle>{title}</AlertTitle> : null}
+        <AlertDescription>{description}</AlertDescription>
+        {link ? <Link href={link.url}>{link.text}</Link> : null}
```

Comment:
I am wondering, should this open in a new tab?

Reply:
Mmmmm I need to doble check with Enrique.

Reply:
Waiting for Enrique's answer

Reply:
As you thought. We need to open it in a new tab. Thanks! (I checked it with Enrique)